src/main/scala/org/broadinstitute/dsde/agora/server/business/AgoraBusiness.scala

```diff
@@ -434,16 +434,16 @@ class AgoraBusiness(permissionsDataSource: PermissionsDataSource)(implicit ec: E
       other than entityType, we'll query mongo first. We exclude entityType because it is too
       broad. The list of parameters below matches what we document in swagger  (minus entityType).
     */
-    def isCriteriaSpecific(agoraSearch: AgoraEntity): Boolean = {
-      agoraSearch.namespace.nonEmpty ||
-        agoraSearch.name.nonEmpty ||
-        agoraSearch.snapshotId.nonEmpty ||
-        agoraSearch.snapshotComment.nonEmpty ||
-        agoraSearch.synopsis.nonEmpty ||
-        agoraSearch.documentation.nonEmpty ||
-        agoraSearch.owner.nonEmpty ||
-        agoraSearch.payload.nonEmpty
-    }
+    // def isCriteriaSpecific(agoraSearch: AgoraEntity): Boolean = {
+    //   agoraSearch.namespace.nonEmpty ||
+    //     agoraSearch.name.nonEmpty ||
+    //     agoraSearch.snapshotId.nonEmpty ||
+    //     agoraSearch.snapshotComment.nonEmpty ||
+    //     agoraSearch.synopsis.nonEmpty ||
+    //     agoraSearch.documentation.nonEmpty ||
+    //     agoraSearch.owner.nonEmpty ||
+    //     agoraSearch.payload.nonEmpty
+    // }
```

Comment:
Why not remove it?

Reply:
That would usually be my call but this particular function was in-progress by David An as recently as last month.

It does actually have a call site at
```
// if (isCriteriaSpecific(agoraSearch)) {
```
that is also commented out with a reasonable explanation - so I decided to comment out the function to match its call site instead of deleting DA's code.